Lib/http/client.py

```diff
@@ -251,7 +255,7 @@ def __init__(self, sock, debuglevel=0, method=None, url=None):
         self.headers = self.msg = None
 
         # from the Status-Line of the response
-        self.version = _UNKNOWN # HTTP-Version
+        self.version = _UNKNOWN  # HTTP-Version
```

Comment:
Avoid changes which aren't relevant to the intent of this pull request. This whitespace change isn't relevant.